pkg/api/handlers/compat/images.go

```diff
@@ -252,6 +252,10 @@ func CreateImageFromImage(w http.ResponseWriter, r *http.Request) {
 
 	fromImage := mergeNameAndTagOrDigest(query.FromImage, query.Tag)
 
+	if shortnames.IsShortName(fromImage) {
```

Comment:
Can you check to see if there is a shortname mapping before you do this.
If someone specifies ubi8, we still want them getting it from redhat not from docker.
If they name is not expandable then this is fine.

Reply:
Call shortname.Resolve(ctx, fromImage) If only one comes back, then use it, otherwise hard code docker.io.

Reply:
Yup, sent a new version of the PR